### DISCUSSION TITLE
Feature: Ability to hide private orgs/repositories in Notifications UI

### DISCUSSION THREAD

**Discussion by NickLarsenNZ** [community/community#37495] [Other Feature Feedback, Questions, & Ideas]
I belong to some private organisations, and in the Notifications view there seems to be no way to hide private organisations/repositories on the left side.

I think there should be an option to hide them from view without identifying them in a filter.

This would be very helpful when doing screensharing/streaming - especially on open-source contributions.

**Comment by ossanna16** [comment]
Hi there @NickLarsenNZ and welcome to our community! Thank you for the feedback 🙂
